templates/model-driven-operations/index.html

```diff
@@ -0,0 +1,473 @@
+            loading="lazy",
+            attrs= {"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/b8e1b994-Onetrail.png",
+            alt="Onetrail",
+            width="350",
+            height="136",
+            hi_def=True,
+            loading="lazy",
+            attrs= {"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/3595713f-barnes-and-noble-logo.png",
+            alt="Barnes and Noble",
+            width="103",
+            height="59",
+            hi_def=True,
+            loading="lazy",
+            attrs= {"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/2616b34f-ACI-Universal-Payments.svg",
+            alt="ACI Universal Payments",
+            width="118",
+            height="54",
+            hi_def=True,
+            loading="lazy",
+            attrs= {"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/cf962284-docapost.png",
+            alt="Docapost",
+            width="800",
+            height="600",
+            hi_def=True,
+            loading="lazy",
+            attrs= {"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/18ea2f90-Tele2_logo.svg",
+            alt="Tele2",
+            width="91",
+            height="35",
+            hi_def=True,
+            loading="lazy",
+            attrs= {"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<hr class="is-fixed-width">
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>JAAS: Managed Juju infrastructure, hosted by Canonical</h2>
+      <p>JAAS is the best way to quickly model and deploy your cloud-based applications. Create, configure and host your applications on the public cloud directly from your browser or the command line.  Concentrate on your software and solutions with a fully managed Juju infrastructure.</p>
+      <p>
+        <a class="p-button--positive p-link--external" href="https://jaas.ai/">Get started with JAAS</a>
+      </p>
+    </div>
+    <div class="col-6">
+      <div class="u-hide--small u-align--center u-vertically-center">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/1dfbab48-jaas.gif",
```

Comment:
This does not work in Firefox on Linux. I just see the `Alt`.

Reply:
Interestingly, this URL seems to work just fine in Firefox mmm https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_1280,h_720/https://assets.ubuntu.com/v1/1dfbab48-jaas.gif

Reply:
Its working on the demo now too!